content/docs/deployment/csmoperator/modules/replication.md

```diff
@@ -5,23 +5,47 @@ description: >
   Pre-requisite for Installing Replication via Dell CSM Operator
 ---
 
-The CSM Replication module for supported Dell CSI Drivers can be installed via the Dell CSM Operator. Dell CSM Operator will deploy CSM Replication sidecar and the complemental CSM Replication controller manager.
+The CSM Replication module for supported Dell CSI Drivers can be installed via the Dell CSM Operator. Dell CSM Operator will deploy the CSM Replication sidecar and the CSM Replication Controller Manager.
 
-## Prerequisite
-
-To use Replication, you need at least two clusters:
+## Prerequisites
+To configure Replication prior to installation via CSM Operator, you need:
 
 - a source cluster which is the main cluster
-- one or more target clusters which will serve as diaster recovery clusters for the main cluster
-
-To configure all the clusters, follow the steps below:
+- a target cluster which will serve as the disaster recovery cluster, a single stretched cluster configuration is also supported
+- the csm-replication GitHub repository is cloned to your source cluster and the [`repctl`](../../../../replication/tools) tool is either built from source or downloaded and in your PATH
 
-1. On your main cluster, follow the instructions available in CSM Replication for [Installation using repctl](../../../../replication/deployment/install-repctl), with the exception of step 4. When you reach step 4, you MUST use the command below to automatically package all clusters `.kube` config as a secret:           
+### Cloning the GitHub Repository and Building repctl
+On your source cluster run the following to clone and build the repctl tool.
 
-```shell
-  ./repctl cluster inject 
+```
+git clone -b v1.4.0 https://github.com/dell/csm-replication.git
+cd csm-replication/repctl
+make build
 ```
 
-CSM Operator needs this admin configs instead of the service accounts’ configs  to be able to properly manage the target clusters. The default service account that'll be used is the CSM Operator service account.
-
-2. On each of the target clusters, configure the prerequisites for deploying the driver via Dell CSM Operator. For example, PowerScale has the following [prerequisites for deploying PowerScale via Dell CSM Operator](../../drivers/powerscale/#prerequisite)
+The rest of the instructions will assume that your current working directory is the csm-replication/repctl directory.
+## Configuration Steps
+To configure replication when using multiple clusters:
```

Comment:
Can this changed to support two cluster?

Reply:
Reworded this as it was from an earlier iteration.